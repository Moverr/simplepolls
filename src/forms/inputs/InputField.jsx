import React from 'react'

const InputField = function (props) {
    return <input {...props} onChange={e => props.callback(e)} />;
}

export default InputField;

