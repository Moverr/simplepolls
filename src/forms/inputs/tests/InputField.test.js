import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
 
import InputField from '../InputField';
 

describe('Testing Input FIeld', () => {
  it('should  render correctly', () => {    
    const props = { };

      const wrapper = shallow(<InputField props />);
      expect(wrapper).toMatchSnapshot();
  });
})



 