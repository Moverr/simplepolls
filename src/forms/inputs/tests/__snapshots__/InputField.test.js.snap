// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Input FIeld should  render correctly 1`] = `
<input
  onChange={[Function]}
  props={true}
/>
`;
